packages/eslint-plugin/src/rules/no-magic-numbers.ts

```diff
@@ -40,7 +40,10 @@ export default util.createRule<Options, MessageIds>({
         },
       },
     ],
-    messages: baseRule.meta.messages,
+    messages: baseRule.meta.messages ?? {
+      useConst: "Number constants declarations must use 'const'.",
+      noMagic: 'No magic number: {{raw}}.',
+    },
```

Comment:
this rule has been using message ids since 5.12.
I guess we should keep this alive until we drop v5.0.0 from the supported ranges.